internal/dyngo/events.go

```diff
@@ -0,0 +1,215 @@
+package dyngo
```

Comment:
Can we move `internal/dyngo` into `internal/appsec/`? I think currently it's tightly coupled with it. Until some other product/feature makes use of it, I think it makes sense. WDYT?

Reply:
No, dyngo is a building block not only for appsec. The profiler and apm will be able to use it from now if needed.

Reply:
If I understood correctly, `dyngo` is the gateway for the rest of the repository to integrate with AppSec functionalities. And because it is specific to AppSec functionalities, it seems only natural to have it inside the `internal/appsec/` package. Today, it is only a utility that the `appsec` package provides. If the profiler or tracer ever need it and it becomes a more generic utility, we can always re-arrange it otherwise or move it, if that turns out making most sense. That is one advantage of the `internal/` approach: you can always re-arrange things without breaking users.

Reply:
I really would like to have dyngo in `internal/` rather than `appsec/` to emphasis the fact everyone can start use it from now. This is not at appsec component. For example, spans could be created out of it.